src/app/jscaip/tests/Direction.spec.ts

```diff
@@ -8,65 +9,67 @@ describe('Direction', () => {
     describe('factory', () => {
         describe('of', () => {
             it('should construct a direction', () => {
-                expect(Direction.factory.of(1, 0)).toBe(Direction.RIGHT);
-                expect(Direction.factory.of(-1, 0)).toBe(Direction.LEFT);
-                expect(Direction.factory.of(0, 1)).toBe(Direction.DOWN);
-                expect(Direction.factory.of(0, -1)).toBe(Direction.UP);
-                expect(Direction.factory.of(1, -1)).toBe(Direction.UP_RIGHT);
-                expect(Direction.factory.of(-1, -1)).toBe(Direction.UP_LEFT);
-                expect(Direction.factory.of(-1, 1)).toBe(Direction.DOWN_LEFT);
-                expect(Direction.factory.of(1, 1)).toBe(Direction.DOWN_RIGHT);
+                // TODO FOR REVIEW: Which one should we choose? .toEqual(success) or .get?
```

Comment:
@MartinREMY42 reste celui-ci

Reply:
Bah, on aura tout les détails qu'il nous faut dans les deux cas (peut être plus avec le .get() vu que MGPFallible a des getter sans avoir leur champ donc les test diront pas "on attendait null et on a eu MESSAGE_D_ERREUR".

Donc goe.
You can take that TODO off.